simulator/docs/extender.md

```diff
@@ -0,0 +1,65 @@
+## Extender
+
+This document describes how to use your `Extenders` in the scheduler running in the simulator.
+The `Extender` is one of the features of the scheduler's webhook on kubernetes.
+
+The simulator stores the results of each Extender in the annotation of a pod.
+
+(Note that it's not related to the [`plugin-extender`](./plugin-extender.md) which is one of the our simulator's feature. 
+(Sorry for the confusing name 😅))
+
+Note: This function uses api-server on the simulator to store the results.
+Therefore, **if the scheduler is not connected to the simulator's api-server,
+this feature is not available.**
```

Comment:
I don't get it. Can you elaborate?

Reply:
Oops, `api-server`→`HTTP server on the simualtor`.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/302/files#diff-512bab2a023bf2e92a0bfdf2995072f49029ae70faa536b01161fa221de09201R11-R13

Reply:
```md
Note: This feature is not available in [external scheduler](./external-scheduler.md).
```